terminus-ui/date-range/src/date-range.component.ts

```diff
@@ -274,6 +278,13 @@ export class TsDateRangeComponent implements OnInit, OnDestroy {
       return;
     }
 
+    this.changeDetectorRef.detectChanges();
+
+    // This is to fix on an initial load, date range value isn't populating correctly.
```

Comment:
Can we prefix this comment with `// HACK: etc..`? Using a standard prefix helps call these specific items out (ie `TODO` etc)